doc/Settings.md

```diff
@@ -373,27 +376,30 @@ By default this will be the one account you create during the installation proce
 But you can expand the list of email addresses by any used email address you want.
 Registration of new accounts with a listed email address is not possible.
 
-    [config]
-    admin_email = you@example.com, buddy@example.com
+	'config' => [
```

Comment:
The indentation is wrong in this line I think.

Reply:
It changed from 4 spaces to a tab.

Reply:
But only in that one line right? The indentation in the diff at github is not consistent.